internal/loglet_test.go

```
@@ -643,3 +648,551 @@
+			t.Errorf("field %q: expected %v, got %v", k, v, fields[k])
+		}
+	}
+}
+
+func testFieldsMapCopyWithExcess(t *testing.T) {
+	var loglet Loglet
+	loglet1 := loglet.WithField("key", testValue)
+
+	testCases := []int{0, 1, 5, 10}
+	for _, excess := range testCases {
+		validateExcessCapacity(t, loglet1, excess)
```

golangci-lint (2.2.2)

copylocks: call of validateExcessCapacity copies lock value: darvaza.org/slog/internal.Loglet contains sync.Once contains sync.noCopy

(govet)